client/static_src/components/reports/reports.vue

```diff
@@ -31,8 +31,10 @@
 
       <popover class="flex-fill mr-2">
         <template slot="popover-button">
-          <icon :icon="['fas', 'filter']" class="mr-1"/>
-          Filter
+          <span  v-bind:class="{active: isFiltered}">
```

Comment:
add a class when a filter is active. the visual feedback (currently bold text) could be better